### PR TITLE
Update to ng-packagr version 5

### PR DESCRIPTION
**Pull request by alan-agius4**
Changelog https://github.com/ng-packagr/ng-packagr/blob/master/CHANGELOG.md#500-2019-03-27

### CODE REVIEW

File: packages/schematics/angular/library/index.ts [hunk 1]
```diff
@@ -103,20 +103,15 @@ function addDependenciesToPackageJson() {
...
+        version: '^5.0.0',
       },
       {
-        type: NodeDependencyType.Dev,
-        name: 'tsickle',
-        version: '>=0.34.0',
-      },
-      {
-        type: NodeDependencyType.Dev,
+        type: NodeDependencyType.Default,
```
Comment (alan-agius4) [inline packages/schematics/angular/library/index.ts]: This is due that by default tslib is a dependency in the workspace `package.json` and we shouldn't add it as both a dev and dependency.

File: packages/angular_devkit/build_ng_packagr/package.json [hunk 2]
```diff
@@ -9,17 +9,15 @@
   "dependencies": {
     "@angular-devkit/architect": "0.0.0",
     "@angular-devkit/core": "0.0.0",
-    "rxjs": "6.4.0",
-    "semver": "5.6.0"
+    "rxjs": "6.4.0"
   },
   "peerDependencies": {
-    "ng-packagr": "^4.0.0"
+    "ng-packagr": "^4.0.0 || 5.0.0"
```
Comment (clydin) [inline packages/angular_devkit/build_ng_packagr/package.json]: peer dependencies should be a range (`^5.0.0`).

Comment (alan-agius4) [reply packages/angular_devkit/build_ng_packagr/package.json]: Oops

**Comment by kyliau** [comment]
@alan-agius4 Could you please take a look at the merge conflicts?

**Comment by alan-agius4** [comment]
@kyliau, rebased.

**Comment by mgechev** [comment]
@kyliau ups, I merged this one before your rotation is over. Sorry.

**Pull request merged**